sources/types.ts

```diff
@@ -5,6 +5,7 @@ export enum SupportedPackageManagers {
   Npm = `npm`,
   Pnpm = `pnpm`,
   Yarn = `yarn`,
+  Cnpm = `cnpm`,
```

Comment:
nit: ASCII/alphabetical order
```suggestion
  Cnpm = `cnpm`,
  Npm = `npm`,
  Pnpm = `pnpm`,
  Yarn = `yarn`,
```